_episodes/12-cluster.md

```diff
@@ -46,6 +47,13 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the internet. We use a program called the Secure SHell or ssh for this. (make sure you have 
```

Comment:
I suggest replacing with `the Secure SHell or ssh` with `a Secure SHell (or SSH) client`.

Reply:
a544bb2

Reply:
a544bb23